strr-strata-web/app/pages/strata-hotel/dashboard/[registrationId].vue

```diff
@@ -37,10 +37,11 @@ onMounted(async () => {
     // existing registration or application under the account
     // set left side of header
     title.value = strataDetails.value.brand.name
+    const nonPlural = strataDetails.value.numberOfUnits === 1
     subtitles.value = [
-      { text: `${strataDetails.value.numberOfUnits} ${t('strr.word.units')}` },
+      { text: `${strataDetails.value.numberOfUnits} ${t('strr.word.unit', nonPlural ? 1 : 2)}` },
```

Comment:
I might be wrong but I think the i18n library will infer this with just 'strataDetails.value.numberOfUnits' instead of 'nonPlural ? 1 : 2'

Reply:
I thought the same thing at first, but for whatever reason it starts at '1' for this one and at '0' for the other one with 3 values